ui/src/components/questions/post.js

```diff
@@ -69,12 +69,26 @@ class Post extends React.Component {
         );
     }
 
-    getTime() {
-        return moment(this.props.created).fromNow();
+    static getTime(localDateTime) {
+        return Post.toMoment(localDateTime)
+            .fromNow();
     }
 
-    getTimeStamp() {
-        return moment(this.props.created).format('llll');
+    static getTimeStamp(localDateTime) {
+        return Post.toMoment(localDateTime)
+            .format('llll');
+    }
+
+    static toMoment(localDateTime) {
+        return moment({
```

Comment:
Are LocalDateTimes returned as objects? If so, that indicates issues with date time formatting from reactive wizard. Expected format is that it's returned as a string.